### ISSUE TITLE
New compatible model: ADEA71A2VEB

### ISSUE THREAD

**Issue by jamofer** [revk/ESP32-Faikin#52]
**Sky Air ADEA71A2VEB**
* Official WiFi module: BRP069C81.
* Ports: X35A (power supply) and X50A (communication).

I've already used it for a week replacing my old BRP069C81 that was failing a lot with Onecta APP.  Faikin Works perfectly!

Here it is my home-assistant screenshot:

![imagen](https://github.com/revk/ESP32-Faikin/assets/9080627/7e760c2b-fa51-4058-aed1-bfcb9cbe970c)

If it is okay, we can update the compatible list Wiki adding this new model.

**Comment by revk** [comment]
It should be directly editable now? Do try.

**Comment by jamofer** [comment]
Done.

Yes it is, but I don't know if I had to discuss it here first before adding it. Thanks!

**Comment by revk** [comment]
Fair enough. As long as people are not being dicks I'm happy for people to update the wiki as needed. So far so good.

**Comment by thanhdatneu** [comment]
Hello, can u show me how to connect with port X35 and x50A ? Thanks

**Comment by revk** [comment]
My air-con came with a lead - not sure if anyone else has pictures.

**Comment by thanhdatneu** [comment]
![image](https://github.com/revk/ESP32-Faikin/assets/16422614/a3ffbcc7-905b-4891-81e9-ce6797ea73a0)

![image](https://github.com/revk/ESP32-Faikin/assets/16422614/d866dd4a-8102-4485-9093-e2e5e03f8743)

This is my x35A and x50A and i don't know how to connect. Tks

**Comment by thanhdatneu** [comment]
![image](https://github.com/revk/ESP32-Faikin/assets/16422614/2d9ca0b7-9a1b-4c58-b933-a94022ddcaed)

![image](https://github.com/revk/ESP32-Faikin/assets/16422614/bf275e5c-7acd-435d-95ae-3c6732e2cfbd)

This is my x35A and x50A and dont know how to connect, Pls check, Many thanks

**Comment by revk** [comment]
The X35A looks like power, the yellow and blue wires here.
The X50A is data, the orange and red wires. The brown is not used.

![IMG_3362](https://github.com/revk/ESP32-Faikin/assets/996983/8cebd243-a80d-431b-81a8-99ee06c91337)
![IMG_3363](https://github.com/revk/ESP32-Faikin/assets/996983/69f448e0-2242-447c-8250-c8294d225d2d)
![IMG_3364](https://github.com/revk/ESP32-Faikin/assets/996983/54f6b746-4cee-45f1-b432-3333cf6d3d1b)

**Comment by thanhdatneu** [comment]
Oh, your port is X50H, It is different from my port, my unit is x50A 😅

**Comment by revk** [comment]
Where do you get X50H from, it's X50A

![IMG_3367](https://github.com/revk/ESP32-Faikin/assets/996983/4a0cb5a8-1a30-4e3f-926e-e37c0b139dfb)

**Comment by thanhdatneu** [comment]
Oh sorry, the image is not clear so i thinh it was "H". I will check my port. Thanks

**Comment by DatutaX** [comment]
Hey, i connect module to my X50A, but it does not work
Wifi ok but system is offline
On X50A the orange pin is 26 or 27 on your module ?

**Comment by revk** [comment]
I'm just about to get on a plane, but see the pictures in this thread.

**Comment by DatutaX** [comment]
@jamofer hello, can u show me your X50A port, thanks

**Comment by revk** [comment]
Literally look at the posts in this discussion, you see the X50 and wiring to it.
